chia/wallet/notification_store.py

```diff
@@ -46,6 +46,8 @@ async def create(
                 "CREATE TABLE IF NOT EXISTS notifications(" "coin_id blob PRIMARY KEY," "msg blob," "amount blob" ")"
             )
 
+            await conn.execute("CREATE TABLE IF NOT EXISTS all_ids(coin_id blob PRIMARY KEY)")
```

Comment:
table name is a bit generic, can we use `notifications_all_ids` or something similar?